okio-zipfilesystem/src/jvmMain/kotlin/okio/zipfilesystem/ZipEntry.kt

```diff
@@ -55,46 +65,48 @@ internal class ZipEntry(
    *
    * @throws IllegalArgumentException if `value` is < 0 or > 0xFFFFFFFFL.
    */
-  val crc: Long,
+  val crc: Long = -1L,
 
   /**
    * Gets the compressed size of this `ZipEntry`.
    *
    * @return the compressed size, or -1 if the compressed size has not been
```

Comment:
nit: inconsistent use of -1 vs `-1`

Reply:
I think -1 is nicer in words, even if it’s -1L in code.

Reply:
Ahh, I see the backticks. Lemme rewrite all these awful comments in a follow up!